web/tests/libtest/codechecker.py

```diff
@@ -96,8 +96,9 @@ def get_diff_results(basenames, newnames, diff_type, format_type=None,
     if extra_args:
         diff_cmd.extend(extra_args)
 
-    out = subprocess.check_output(
-        diff_cmd, encoding="utf-8", errors="ignore", env=cc_env)
+    out, _ = subprocess.Popen(
```

Comment:
Why do we have to change the original code here?

Reply:
We had to change this, because `check_output()` throws an exception in case of non-zero exit code. In this commit we changed exit code to 2 if `CodeChecker cmd diff` results some difference in reports. However, even in this case we'd like to check the standard output and standard error.